src/cmd/src/cli/export.rs

```diff
@@ -226,7 +226,7 @@ impl Export {
     }
 
     async fn show_create_table(&self, catalog: &str, schema: &str, table: &str) -> Result<String> {
-        let sql = format!("show create table {}.{}.{}", catalog, schema, table);
+        let sql = format!("show create table '{}'.'{}'.'{}'", catalog, schema, table);
```

Comment:
Looks like we should use double quote or escape quote here?

Reply:
IIUC, these three names are `the value of the identifier without quotes.`(https://docs.rs/sqlparser/latest/sqlparser/ast/struct.Ident.html#structfield.value)

Reply:
@dimbtp do you test this function work well locally?

@killme2008 means SQL standard to quote identifier is by `"` instead of `'`. See https://stackoverflow.com/questions/2901453/sql-standard-to-escape-column-names.

Reply:
I noticed this issue before, but when using `mysql` connect to the greptimedb engine, I can freely use any quote style to execute the sql. And I notice that https://github.com/GreptimeTeam/greptimedb/blob/c00c1d95eed0b8e8035a206256207299fabd887d/src/cmd/src/cli/export.rs#L171-L177 use single quote, so I follow the same rule.

> do you test this function work well locally?

I test it locally and it works.

Reply:
Thanks for your info! I checked the underneath code in sqlparser-rs:

```rust
    /// Parse a simple one-word identifier (possibly quoted, possibly a keyword)
    pub fn parse_identifier(&mut self) -> Result<Ident, ParserError> {
        let next_token = self.next_token();
        match next_token.token {
            Token::Word(w) => Ok(w.to_ident()),
            Token::SingleQuotedString(s) => Ok(Ident::with_quote('\'', s)),
            Token::DoubleQuotedString(s) => Ok(Ident::with_quote('\"', s)),
            _ => self.expected("identifier", next_token),
        }
    }
```

It seems both single quote and double quote are supported, while it may not be quite identical with SQL standarad.

Reply:
But using double quoted string would be more "formal". The example you refer to is about SQL string, not identifier.

I don't insist on this, although.

Reply:
@MichaelScofield According to https://github.com/GreptimeTeam/greptimedb/blob/c00c1d95eed0b8e8035a206256207299fabd887d/src/cmd/src/cli.rs#L125-L130, `repl` function should wait a little bit as I don't observe too much progress on #2373

I have made a local test and prepare to push it for review.

Reply:
@dimbtp we can make repl simple in that it does not need to talk to datanode, just frontend is enough. Cut down some functionalities. So the work is down to make it connect to frontend and submit the sql through its grpc client. Are you willing to take #2373?

Reply:
I can see that the conversation is far beyond the scope of this patch.

If we agree that this patch is correct, I suggest we merge it and leave further test coverage or even improvement of test utils to follow-ups where @dimbtp probably world like to pick up.

```suggestion
        let sql = format!(r#"show create table "{}"."{}"."{}""#, catalog, schema, table);
```

I'd formally suggest this change to avoid some tricky coincides.